sql/core/src/main/scala/org/apache/spark/sql/execution/SparkQl.scala

```diff
@@ -62,6 +66,458 @@ private[sql] class SparkQl(conf: ParserConf = SimpleParserConf()) extends Cataly
         val tableIdent = extractTableIdent(nameParts)
         RefreshTable(tableIdent)
 
+      case Token("TOK_CREATEDATABASE", Token(databaseName, Nil) :: createDatabaseArgs) =>
```

Comment:
cc @hvanhovell 

any suggestions on how we can make this file/function more modular? It is getting too long and we are about to add a lot more statements to it.